### ISSUE TITLE
iOS 5.1.7 / The operation could not be completed. Capacitor.CapacitorUrlRequest

### ISSUE THREAD

**Issue by rob365me** [floccusaddon/floccus#1629]
### Which version of floccus are you using?

5.1.7

### How many bookmarks do you have, roughly?

400

### Are you using other means to sync bookmarks in parallel to floccus?

No

### Sync method

Git

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

iOS 17.5.1 iPhone 12 Pro Max

### Describe the Bug

As soon as I create a profile on the iOS app (I have the extensions syncing successfully to a git repo with these settings) I get the error in the attached screenshot. 
![20240530_003452000_iOS](https://github.com/floccusaddon/floccus/assets/47661361/932f9acf-bbdd-4810-84dd-0f9cedc1815e)

### Expected Behavior

Sync bookmarks.

### To Reproduce

Install from App Store.  Configure for git over https.  Witness error. 

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
Hi @rob365me 
Thank you for reporting this. This is a known issue with the native app, that will hopefully be fixed soon. The bug is in Capacitor the mobile framework we use.

**Comment by marcelklehr** [comment]
Duplicate of #1639
